Jenkinsfile

```diff
@@ -166,6 +166,12 @@ try {
             stage("Linting") {
               sh "scripts/lint.sh"
             }
+          }, nix: {
```

Comment:
Now that is runs: can we move it to the very last stage and add a debug execution of `hyriseSystemsTest`? (we might need to check for the typical exclusions)

Reply:
I probably wasn't clear here. I meant the last linux stage where valgrind etc. runs.
The linux setup runs in Docker, so I am fine with nix there. No way we let this run on the macOS nodes. ;)